Resolve a bug in GraphQL

MathieuDutSik opened the pull request:
The problem is the following. The following code does not compile
```rust
struct TestView<C> {
  view: RegisterView<C, String>,
}

#[derive(GraphQLView)]
struct BlockView<C> {
  bids: CollectionView<C,String, TestView<C>>,
  asks: CollectionView<C,String, TestView<C>>,
}
```
The reason is that the code for `TestViewEntry` is generated two times.
The solution is to generate two codes: `TestViewBlockViewbidsEntry` and `TestViewBlockViewasksEntry` so as to avoid the namespace collision.

This is the only solution that makes sense. This is because the generated code refers both to the key and to the value. So, we cannot generate a `TestViewEntry` at the level of the `TestView`.

ma2bd commented:
Good catch. Since the field name is unique in each container, I would suggest dropping the type. Also please use CamelCase: `BlockViewBidsEntry` instead of `TestViewBlockViewbidsEntry`.

MathieuDutSik commented:
> Good catch. Since the field name is unique in each container, I would suggest dropping the type. Also please use CamelCase: `BlockViewBidsEntry` instead of `TestViewBlockViewbidsEntry`.

Ok, I corrected that (and rebased).

ma2bd commented on linera-views-derive/src/util.rs:
```diff
@@ -10,9 +10,18 @@ pub fn snakify(r#type: &Type) -> Ident {
-pub fn create_entry_name(r#type: &Type) -> Ident {
-    transform_type_to_ident(r#type, |s: String| format!("{}{}", s, "Entry"))
+/// Capitalize the first character of a string
+pub fn capitalize_first_character(s: &str) -> String {
```
I guess this works for now but converting from snake_case to CamlCase is actually more complicated than that.
In serde-generate, I used `heck`: https://crates.io/crates/heck

MathieuDutSik replied:
Thanks, I modify to use `heck`.
